src/Miunie.Discord/Adapters/DiscordGuildsAdapter.cs

```diff
@@ -16,11 +16,11 @@ public DiscordGuildsAdapter(IDiscord discord, EntityConvertor entityConvertor)
             _entityConvertor = entityConvertor;
         }
 
-        public async Task<MiunieGuild> FromAsync(MiunieUser user)
+        public Task<MiunieGuild> FromAsync(MiunieUser user)
```

Comment:
Method isn't async but has the suffix Async, it is a Task though so 🤷‍♂

Reply:
Async suffix is used to display that the method can be run asynchronously. FromAsync is the correct format.